drivers/vdpa/virtio/rte_vf_rpc.c

```diff
@@ -43,7 +43,9 @@ rte_vdpa_vf_dev_add(char *vf_name, const char *vm_uuid,
 		offset = snprintf(args, RTE_DEV_NAME_MAX_LEN, "%s=%s", VIRTIO_ARG_VDPA, VIRTIO_ARG_VDPA_VALUE_VF);
 
 	if (stage1)
-		snprintf(args + offset, RTE_DEV_NAME_MAX_LEN, ",%s=%s", VIRTIO_ARG_VDPA_STAGE, VIRTIO_ARG_VDPA_VALUE_STAGE);
+		offset += snprintf(args + offset, RTE_DEV_NAME_MAX_LEN, ",%s=%s", VIRTIO_ARG_VDPA_STAGE, VIRTIO_ARG_VDPA_VALUE_STAGE);
+
+	offset += snprintf(args + offset, MAX_PATH_LEN, ",%s=%s", VIRTIO_ARG_VDPA_SOCK_PATH, ifname);
```

Comment:
this 'offset +=' seems not needed as offset is not used later

Reply:
just in case we add another line , needed add + again.